Mathlib/LinearAlgebra/Matrix/Orthogonal.lean

```diff
@@ -17,10 +17,10 @@ This file contains definitions and properties concerning orthogonality of rows a
 
 ## Main results
 
-- `matrix.has_orthogonal_rows`:
-  `A.has_orthogonal_rows` means `A` has orthogonal (with respect to `dot_product`) rows.
-- `matrix.has_orthogonal_cols`:
-  `A.has_orthogonal_cols` means `A` has orthogonal (with respect to `dot_product`) columns.
+- `Matrix.OrthogonalRows`:
+  `A.OrthogonalRows` means `A` has orthogonal (with respect to `dotProduct`) rows.
+- `Matrix.OrthogonalCols`:
+  `A.OrthogonalCols` means `A` has orthogonal (with respect to `dotProduct`) columns.
```

Comment:
I think we should keep the `Has` prefix, since this isn't a typeclass.

Reply:
If you disagree, or don't like either option, feel free to get more opinions on Zulip.